src/main/java/org/spongepowered/common/mixin/entitycollisions/MixinBlock_Collisions.java

```diff
@@ -82,7 +83,8 @@ public boolean requiresCollisionsCacheRefresh() {
     @Override
     public void initializeCollisionState(World worldIn) {
         SpongeConfig<? extends GeneralConfigBase> activeConfig = ((IMixinWorldServer) worldIn).getActiveConfig();
-        EntityCollisionCategory collisionCat = activeConfig.getConfig().getEntityCollisionCategory();
+        SpongeConfig<? extends GeneralConfigBase> globalConfig = SpongeImpl.getGlobalConfig();
+        EntityCollisionCategory collisionCat = globalConfig.getConfig().getEntityCollisionCategory();
         this.setMaxCollisions(collisionCat.getMaxEntitiesWithinAABB());
```

Comment:
The value should still be read from the `activeConfig` instance, to allow for per-world overrides.